services/cert-manager/0.2.7/root-ca/root-ca.yaml

```diff
@@ -4,36 +4,33 @@
 ## follow-up ticket for mirroring: https://jira.d2iq.com/browse/D2IQ-78135
 ---
 apiVersion: cert-manager.io/v1
-kind: Issuer
+kind: ClusterIssuer
 metadata:
-  name: kommander-bootstrap-ca-issuer
-  namespace: ${releaseNamespace}
+  name: selfsigned-issuer
 spec:
   selfSigned: {}
 ---
 apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
-  name: kommander-bootstrap-root-certificate
-  namespace: ${releaseNamespace}
+  name: kommander-ca
```

Comment:
on my local cluster, I see this Certificate and ClusterIssuer already exist (not sure who creates them). Are these only created on attached clusters?

Reply:
The resources in this files will only be created in attached clusters.

On the management host, these 3 resources are already created by [kommander-cli](https://github.com/mesosphere/kommander-cli/tree/main/pkg/installer/manifests/rootca).